theme/sass/components/_card-component.scss

```diff
@@ -1,101 +1,126 @@
 .card-component-container {
-    width: 100%;
-    margin: 0 auto;
-    display: block;
-    padding: 0 2rem;
+  width: 100%;
+  margin: 0 auto;
+  display: block;
+  padding: 0 2rem;
+
+  @include breakpoint($md) {
+    width: $gutter-size;
+    padding: 0;
+  }
+
+  h2 {
```

Comment:
Remove top margin because gap is too big due to the padding on the text container